cmd/root.go

```diff
@@ -74,75 +68,55 @@ func PreRun(cmd *cobra.Command, args []string) {
 		})
 	}
 
-	if enabled, _ := f.GetBool("debug"); enabled {
+	if viper.GetBool("debug") {
 		log.SetLevel(log.DebugLevel)
 	}
-	if enabled, _ := f.GetBool("trace"); enabled {
+	if viper.GetBool("trace") {
 		log.SetLevel(log.TraceLevel)
 	}
 
-	pollingSet := f.Changed("interval")
-	schedule, _ := f.GetString("schedule")
-	cronLen := len(schedule)
+	// Update schedule if interval helper is set
+	if viper.IsSet("interval") {
+		if viper.IsSet("schedule") {
+			log.Fatal("only schedule or interval can be defined, not both")
+		}
+		interval := viper.GetInt("interval")
+		viper.Set("schedule", fmt.Sprintf("@every %ds", interval))
+	}
```

Comment:
This part does not work correctly, as "interval" has a default value. Should be easy enough to fix though.

Reply:
Fixed!